ui/ducks/send/send.js

```diff
@@ -1905,6 +1909,16 @@ export function updateSendAsset(
           error: null,
         }),
       );
+
+      // If there is an unapprovedTx in background state and its type was a token method,
+      // then the user will not want to send any hex data now that they have change the
+      // transaction from being a token transfer to a simple send.
+      if (
+        unapprovedTx?.type === TRANSACTION_TYPES.TOKEN_METHOD_TRANSFER_FROM ||
+        unapprovedTx?.type === TRANSACTION_TYPES.TOKEN_METHOD_TRANSFER
```

Comment:
I know this is nitty, but when I read this comment I thought unapprovedTx was like...*any* unapproved tx. Can we change this comment to be something along the lines of if the original type of this transaction was a token method then we need to null out he usetInputHeadData (words are not important just specifying original and ensuring that the unapprovedTx is *clearly* marked as the original tx in an edit flow)

Reply:
Comment improved in 91d8a85d2

Reply:
Should we include `TOKEN_METHOD_SAFE_TRANSFER_FROM` here. I don't believe it's possible for it to be a problem at the moment because it's a method on ERC721's and we don't enable the `edit` button on ERC721 sends at the moment, but we may want to soon?

Reply:
added in 1eed22c1d